lib/web_ui/lib/src/engine/browser_detection.dart

```diff
@@ -121,3 +121,19 @@ OperatingSystem _detectOperatingSystem() {
     return OperatingSystem.unknown;
   }
 }
+
+/// List of Operating Systems we know to be working on laptops/desktops.
+///
+/// These devices tend to behave differently on many core issues such as events,
+/// screen readers, input devices.
+List<OperatingSystem> _desktopOperatingSystems = [
```

Comment:
Use a Set instead of a list, and make it const

Reply:
Done! Thanks!